cmd/pipeline/config.go

```diff
@@ -175,6 +176,7 @@ func configure(v *viper.Viper, p *pflag.FlagSet) {
 	v.SetDefault("pipeline::caCertFile", "")
 	v.SetDefault("pipeline::certFile", "")
 	v.SetDefault("pipeline::keyFile", "")
+	v.SetDefault("pipeline::commonName", "pipeline")
```

Comment:
What's the benefit of setting the common name? IMHO we probably don't even have to validate the common name. Matching the CA is more than enough, isn't it?

Reply:
Fixed.